pkg/kubelet/dockershim/naming_test.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	"github.com/stretchr/testify/assert"
 
-	runtimeapi "k8s.io/kubernetes/pkg/kubelet/apis/cri/runtime/v1alpha2"
+	runtimeapi "k8s.io/cri-api/pkg/apis/cri/runtime/v1alpha2"
```

Comment:
Before moving this into stages, I think it's better rename the package back to something like `cri/v1alpha2/runtime`. Or else, for every version updates, the package name would different and hard to keep backward compatible.

Reply:
@feiskyer please see the existing pattern, the version numbers are all at the end.

```
./src/k8s.io/csi-api/pkg/apis/csi/v1alpha1
./src/k8s.io/csi-api/pkg/client/informers/externalversions/csi/v1alpha1
./src/k8s.io/csi-api/pkg/client/listers/csi/v1alpha1
./src/k8s.io/csi-api/pkg/client/clientset/versioned/typed/csi/v1alpha1
./src/k8s.io/metrics/pkg/apis/metrics/v1beta1
./src/k8s.io/metrics/pkg/apis/metrics/v1alpha1
./src/k8s.io/metrics/pkg/apis/custom_metrics/v1beta2
./src/k8s.io/metrics/pkg/apis/custom_metrics/v1beta1
./src/k8s.io/metrics/pkg/apis/external_metrics/v1beta1
./src/k8s.io/metrics/pkg/client/clientset/versioned/typed/metrics/v1beta1
./src/k8s.io/metrics/pkg/client/clientset/versioned/typed/metrics/v1alpha1
./src/k8s.io/client-go/informers/settings/v1alpha1
./src/k8s.io/client-go/informers/rbac/v1
./src/k8s.io/client-go/informers/rbac/v1beta1
./src/k8s.io/client-go/informers/rbac/v1alpha1
./src/k8s.io/client-go/informers/core/v1
./src/k8s.io/client-go/informers/coordination/v1
./src/k8s.io/client-go/informers/coordination/v1beta1
./src/k8s.io/client-go/informers/scheduling/v1
./src/k8s.io/client-go/informers/scheduling/v1beta1
./src/k8s.io/client-go/informers/scheduling/v1alpha1
./src/k8s.io/client-go/informers/auditregistration/v1alpha1
./src/k8s.io/client-go/informers/networking/v1
./src/k8s.io/client-go/informers/networking/v1beta1
./src/k8s.io/client-go/informers/certificates/v1beta1
./src/k8s.io/client-go/informers/storage/v1
./src/k8s.io/client-go/informers/storage/v1beta1
./src/k8s.io/client-go/informers/storage/v1alpha1
./src/k8s.io/client-go/informers/extensions/v1beta1
./src/k8s.io/client-go/informers/batch/v1
./src/k8s.io/client-go/informers/batch/v1beta1
./src/k8s.io/client-go/informers/admissionregistration/v1beta1
./src/k8s.io/client-go/informers/node/v1beta1
./src/k8s.io/client-go/informers/node/v1alpha1
./src/k8s.io/client-go/informers/events/v1beta1
./src/k8s.io/client-go/informers/autoscaling/v1
./src/k8s.io/client-go/informers/apps/v1
./src/k8s.io/client-go/informers/apps/v1beta2
./src/k8s.io/client-go/informers/apps/v1beta1
./src/k8s.io/client-go/informers/policy/v1beta1
./src/k8s.io/client-go/listers/settings/v1alpha1
./src/k8s.io/client-go/listers/rbac/v1
./src/k8s.io/client-go/listers/rbac/v1beta1
./src/k8s.io/client-go/listers/rbac/v1alpha1
./src/k8s.io/client-go/listers/core/v1
./src/k8s.io/client-go/listers/coordination/v1
./src/k8s.io/client-go/listers/coordination/v1beta1
./src/k8s.io/client-go/listers/scheduling/v1
./src/k8s.io/client-go/listers/scheduling/v1beta1
./src/k8s.io/client-go/listers/scheduling/v1alpha1
./src/k8s.io/client-go/listers/auditregistration/v1alpha1
./src/k8s.io/client-go/listers/networking/v1
./src/k8s.io/client-go/listers/networking/v1beta1
./src/k8s.io/client-go/listers/certificates/v1beta1
./src/k8s.io/client-go/listers/storage/v1
./src/k8s.io/client-go/listers/storage/v1beta1
./src/k8s.io/client-go/listers/storage/v1alpha1
./src/k8s.io/client-go/listers/extensions/v1beta1
./src/k8s.io/client-go/listers/batch/v1
./src/k8s.io/client-go/listers/batch/v1beta1
./src/k8s.io/client-go/listers/admissionregistration/v1beta1
./src/k8s.io/client-go/listers/node/v1beta1
./src/k8s.io/client-go/listers/node/v1alpha1
./src/k8s.io/client-go/listers/events/v1beta1
./src/k8s.io/client-go/listers/autoscaling/v1
./src/k8s.io/client-go/listers/authorization/v1
./src/k8s.io/client-go/listers/authorization/v1beta1
./src/k8s.io/client-go/listers/apps/v1
./src/k8s.io/client-go/listers/apps/v1beta2
./src/k8s.io/client-go/listers/apps/v1beta1
./src/k8s.io/client-go/listers/authentication/v1
./src/k8s.io/client-go/listers/authentication/v1beta1
./src/k8s.io/client-go/listers/imagepolicy/v1alpha1
./src/k8s.io/client-go/listers/policy/v1beta1
./src/k8s.io/client-go/tools/clientcmd/api/v1
./src/k8s.io/client-go/kubernetes/typed/settings/v1alpha1
./src/k8s.io/client-go/kubernetes/typed/rbac/v1
./src/k8s.io/client-go/kubernetes/typed/rbac/v1beta1
./src/k8s.io/client-go/kubernetes/typed/rbac/v1alpha1
./src/k8s.io/client-go/kubernetes/typed/core/v1
./src/k8s.io/client-go/kubernetes/typed/coordination/v1
./src/k8s.io/client-go/kubernetes/typed/coordination/v1beta1
./src/k8s.io/client-go/kubernetes/typed/scheduling/v1
./src/k8s.io/client-go/kubernetes/typed/scheduling/v1beta1
./src/k8s.io/client-go/kubernetes/typed/scheduling/v1alpha1
./src/k8s.io/client-go/kubernetes/typed/auditregistration/v1alpha1
./src/k8s.io/client-go/kubernetes/typed/networking/v1
./src/k8s.io/client-go/kubernetes/typed/networking/v1beta1
./src/k8s.io/client-go/kubernetes/typed/certificates/v1beta1
./src/k8s.io/client-go/kubernetes/typed/storage/v1
./src/k8s.io/client-go/kubernetes/typed/storage/v1beta1
./src/k8s.io/client-go/kubernetes/typed/storage/v1alpha1
./src/k8s.io/client-go/kubernetes/typed/extensions/v1beta1
./src/k8s.io/client-go/kubernetes/typed/batch/v1
./src/k8s.io/client-go/kubernetes/typed/batch/v1beta1
./src/k8s.io/client-go/kubernetes/typed/admissionregistration/v1beta1
./src/k8s.io/client-go/kubernetes/typed/node/v1beta1
./src/k8s.io/client-go/kubernetes/typed/node/v1alpha1
./src/k8s.io/client-go/kubernetes/typed/events/v1beta1
./src/k8s.io/client-go/kubernetes/typed/autoscaling/v1
./src/k8s.io/client-go/kubernetes/typed/authorization/v1
./src/k8s.io/client-go/kubernetes/typed/authorization/v1beta1
./src/k8s.io/client-go/kubernetes/typed/apps/v1
./src/k8s.io/client-go/kubernetes/typed/apps/v1beta2
./src/k8s.io/client-go/kubernetes/typed/apps/v1beta1
./src/k8s.io/client-go/kubernetes/typed/authentication/v1
./src/k8s.io/client-go/kubernetes/typed/authentication/v1beta1
./src/k8s.io/client-go/kubernetes/typed/policy/v1beta1
./src/k8s.io/client-go/pkg/apis/clientauthentication/v1beta1
./src/k8s.io/client-go/pkg/apis/clientauthentication/v1alpha1
./src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1
./src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1
./src/k8s.io/kube-aggregator/pkg/client/informers/externalversions/apiregistration/v1
./src/k8s.io/kube-aggregator/pkg/client/informers/externalversions/apiregistration/v1beta1
./src/k8s.io/kube-aggregator/pkg/client/listers/apiregistration/v1
./src/k8s.io/kube-aggregator/pkg/client/listers/apiregistration/v1beta1
./src/k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/typed/apiregistration/v1
./src/k8s.io/kube-aggregator/pkg/client/clientset_generated/clientset/typed/apiregistration/v1beta1
./src/k8s.io/cri-api/pkg/apis/cri/runtime/v1alpha2
./src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/apis/cr/v1
./src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/client/informers/externalversions/cr/v1
./src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/client/listers/cr/v1
./src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/client/clientset/versioned/typed/cr/v1
./src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1
./src/k8s.io/apiextensions-apiserver/pkg/client/informers/externalversions/apiextensions/v1beta1
./src/k8s.io/apiextensions-apiserver/pkg/client/listers/apiextensions/v1beta1
./src/k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1beta1
./src/k8s.io/kube-controller-manager/config/v1alpha1
./src/k8s.io/sample-controller/pkg/generated/informers/externalversions/samplecontroller/v1alpha1
./src/k8s.io/sample-controller/pkg/generated/listers/samplecontroller/v1alpha1
./src/k8s.io/sample-controller/pkg/generated/clientset/versioned/typed/samplecontroller/v1alpha1
./src/k8s.io/sample-controller/pkg/apis/samplecontroller/v1alpha1
./src/k8s.io/kube-scheduler/config/v1alpha1
./src/k8s.io/kubelet/config/v1beta1
./src/k8s.io/node-api/pkg/apis/node/v1alpha1
./src/k8s.io/node-api/pkg/client/informers/externalversions/node/v1alpha1
./src/k8s.io/node-api/pkg/client/listers/node/v1alpha1
./src/k8s.io/node-api/pkg/client/clientset/versioned/typed/node/v1alpha1
./src/k8s.io/kube-proxy/config/v1alpha1
./src/k8s.io/api/settings/v1alpha1
./src/k8s.io/api/rbac/v1
./src/k8s.io/api/rbac/v1beta1
./src/k8s.io/api/rbac/v1alpha1
./src/k8s.io/api/core/v1
./src/k8s.io/api/coordination/v1
./src/k8s.io/api/coordination/v1beta1
./src/k8s.io/api/scheduling/v1
./src/k8s.io/api/scheduling/v1beta1
./src/k8s.io/api/scheduling/v1alpha1
./src/k8s.io/api/auditregistration/v1alpha1
./src/k8s.io/api/networking/v1
./src/k8s.io/api/networking/v1beta1
./src/k8s.io/api/admission/v1beta1
./src/k8s.io/api/certificates/v1beta1
./src/k8s.io/api/storage/v1
./src/k8s.io/api/storage/v1beta1
./src/k8s.io/api/storage/v1alpha1
./src/k8s.io/api/extensions/v1beta1
./src/k8s.io/api/batch/v1
./src/k8s.io/api/batch/v1beta1
./src/k8s.io/api/admissionregistration/v1beta1
./src/k8s.io/api/node/v1beta1
./src/k8s.io/api/node/v1alpha1
./src/k8s.io/api/events/v1beta1
./src/k8s.io/api/autoscaling/v1
./src/k8s.io/api/authorization/v1
./src/k8s.io/api/authorization/v1beta1
./src/k8s.io/api/apps/v1
./src/k8s.io/api/apps/v1beta2
./src/k8s.io/api/apps/v1beta1
./src/k8s.io/api/authentication/v1
./src/k8s.io/api/authentication/v1beta1
./src/k8s.io/api/imagepolicy/v1alpha1
./src/k8s.io/api/policy/v1beta1
./src/k8s.io/apiserver/pkg/apis/example2/v1
./src/k8s.io/apiserver/pkg/apis/config/v1
./src/k8s.io/apiserver/pkg/apis/example/v1
./src/k8s.io/apiserver/pkg/apis/audit/v1
./src/k8s.io/apiserver/pkg/apis/audit/v1beta1
./src/k8s.io/apiserver/pkg/apis/audit/v1alpha1
./src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1
./src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/apis/webhookadmission/v1alpha1
./src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1
./src/k8s.io/sample-apiserver/pkg/generated/informers/externalversions/wardle/v1beta1
./src/k8s.io/sample-apiserver/pkg/generated/informers/externalversions/wardle/v1alpha1
./src/k8s.io/sample-apiserver/pkg/generated/listers/wardle/v1beta1
./src/k8s.io/sample-apiserver/pkg/generated/listers/wardle/v1alpha1
./src/k8s.io/sample-apiserver/pkg/generated/clientset/versioned/typed/wardle/v1beta1
./src/k8s.io/sample-apiserver/pkg/generated/clientset/versioned/typed/wardle/v1alpha1
./src/k8s.io/sample-apiserver/pkg/apis/wardle/v1beta1
./src/k8s.io/sample-apiserver/pkg/apis/wardle/v1alpha1
./src/k8s.io/code-generator/_examples/MixedCase/informers/externalversions/example/v1
./src/k8s.io/code-generator/_examples/MixedCase/listers/example/v1
./src/k8s.io/code-generator/_examples/MixedCase/apis/example/v1
./src/k8s.io/code-generator/_examples/MixedCase/clientset/versioned/typed/example/v1
./src/k8s.io/code-generator/_examples/crd/informers/externalversions/example2/v1
./src/k8s.io/code-generator/_examples/crd/informers/externalversions/example/v1
./src/k8s.io/code-generator/_examples/crd/listers/example2/v1
./src/k8s.io/code-generator/_examples/crd/listers/example/v1
./src/k8s.io/code-generator/_examples/crd/apis/example2/v1
./src/k8s.io/code-generator/_examples/crd/apis/example/v1
./src/k8s.io/code-generator/_examples/crd/clientset/versioned/typed/example2/v1
./src/k8s.io/code-generator/_examples/crd/clientset/versioned/typed/example/v1
./src/k8s.io/code-generator/_examples/apiserver/informers/externalversions/example2/v1
./src/k8s.io/code-generator/_examples/apiserver/informers/externalversions/example/v1
./src/k8s.io/code-generator/_examples/apiserver/listers/example2/v1
./src/k8s.io/code-generator/_examples/apiserver/listers/example/v1
./src/k8s.io/code-generator/_examples/apiserver/apis/example2/v1
./src/k8s.io/code-generator/_examples/apiserver/apis/example/v1
./src/k8s.io/code-generator/_examples/apiserver/clientset/versioned/typed/example2/v1
./src/k8s.io/code-generator/_examples/apiserver/clientset/versioned/typed/example/v1
./src/k8s.io/apimachinery/pkg/apis/testapigroup/v1
./src/k8s.io/apimachinery/pkg/apis/meta/v1
./src/k8s.io/apimachinery/pkg/apis/meta/v1beta1
./src/k8s.io/component-base/config/v1alpha1
```

Reply:
Keeping version at the end is good.

Reply:
OK, then let's keep this pattern.